extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -446,6 +446,12 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @override */
   resumeCallback() {
+    // FIE that was not destroyed on unlayoutCallback does not require a new
+    // ad request.
+    if (!isExperimentOn(this.win, 'a4a-fie-unlayout-enabled') &&
```

Comment:
No need unless we want/need to disable this new feature.  This was added so that we could easily disable if needed.